libs/react-helpers/src/hooks/use-mutation-observer.ts

```diff
@@ -34,7 +33,8 @@ export function useMutationObserver(
     try {
       observer.observe(target, options.config);
     } catch (err) {
-      captureException(err);
+      console.error("Can't observe mutation target", err);
```

Comment:
I think lets just remove the entire try catch, we'll let Sentry pick up and log the error